specification/sql/resource-manager/Microsoft.Sql/preview/2022-02-01-preview/ManagedInstanceDTCs.json

```diff
@@ -0,0 +1,305 @@
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidDtcAuthenticationMethod - DTC authentication method \"{0}\" is not supported.\n\n * 400 ManagementServiceManagedInstanceDtcFeatureDisabled - User tried to interact with Azure SQL Managed Instance DTC settings but DTC feature is currently not supported.\n\n * 400 InvalidDtcVirtualClusterVersion - Virtual cluster version of the Azure SQL Managed Instance \"{0}\" is not supported.\n\n * 404 ResourceNotFound - The requested resource was not found.\n\n * 404 OperationIdNotFound - The operation with Id does not exist.\n\n * 404 OperationIdNotFound - The operation with Id does not exist.\n\n * 409 OperationCancelled - The operation has been cancelled by user.\n\n * 409 OperationInterrupted - The operation on the resource could not be completed because it was interrupted by another operation on the same resource.\n\n * 500 OperationTimedOut - The operation timed out and automatically rolled back. Please retry the operation."
+          },
+          "202": {
+            "description": "Applying DTC settings is in progress"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Updates the managed instance DTC settings with all optional parameters specified.": {
+            "$ref": "./examples/ManagedInstanceDtcUpdateMax.json"
+          },
+          "Updates the managed instance DTC settings by enabling DTC.": {
+            "$ref": "./examples/ManagedInstanceDtcUpdateEnableDtc.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ManagedInstanceDTC": {
+      "description": "SQL Managed Instance DTC",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ManagedInstanceDTCProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ManagedInstanceDTCListResult": {
+      "description": "A list of managed instance's DTCs.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedInstanceDTC"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ManagedInstanceDTCProperties": {
+      "description": "The properties of managed instance DTC.",
+      "type": "object",
+      "properties": {
+        "dtcEnabled": {
+          "description": "Active status of managed instance DTC.",
+          "type": "boolean"
+        },
```

Comment:
consider updating these boolean values to enum as explained here : 

[https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html)

"Replace boolean/switch properties with better enum"

Reply:
I've replied to the other comment, please see the reply there.